Makefile.am

```diff
@@ -18,6 +18,21 @@ service_in_files = \
 	data/com.endlessm.DiscoveryFeed.service.in \
         $(NULL)
 
+gsettings_SCHEMAS = data/com.endlessm.DiscoveryFeed.gschema.xml
```

Comment:
Borrowed from the shell. It does work here, though if there is something I miss, please let me know.

Reply:
Are we calling @GSETTINGS_RULES@ in the makefile?

Reply:
Ah, we are. All good